core/src/main/java/org/apache/carbondata/core/constants/CarbonCommonConstants.java

```diff
@@ -1744,89 +2021,101 @@ private CarbonCommonConstants() {
    * BYTE_ENCODING
    */
   public static final String BYTE_ENCODING = "ISO-8859-1";
+
```

Comment:
carbon.push.rowfilters.for.vector is not in correct place and it should be dynamic=true

Reply:
move it to Query parameter, we can support dynamic=true for it in a new PR. There are others parameters like that.

Reply:
support configure carbon.push.rowfilters.for.vector in set command, dynamic configurable in this PR